sdks/java/io/azure/src/main/java/org/apache/beam/sdk/io/azure/blobstore/AzureBlobStoreFileSystem.java

```diff
@@ -68,6 +68,8 @@
   private static final ImmutableSet<String> NON_READ_SEEK_EFFICIENT_ENCODINGS =
       ImmutableSet.of("gzip");
 
+  private static final int expiryTime = 86400000;
```

Comment:
Constants usually have uppercase names, like so:

```suggestion
  private static final int DEFAULT_EXPIRY_TIME = 86400000;
```